pages/importance.md

```diff
@@ -63,32 +63,31 @@ Thus, the purpose of the guide is to teach how to protect publicly available inf
+*Interesting case study: [in 2012, there was a high-profile case](https://habr.com/ru/post/149179/) of hijacking 4 accounts of a person with only address, name and email. An elaborate access recovery chain - first using available data, then linking fake data - allowed one to first gain access to Amazon, Gmail, Apple, and Twitter, and later even remotely erase data from a person's devices.*
 
 ---
 
-Возможно, вас смутил пункт "биометрические данные", но, увы, их использование уже давно вошло в нашу жизнь. Уже давно распространены средства поиска по лицу, которые используют те же технологии, что и для разблокировки личного телефона через переднюю камеру, а также обширные базы данных из социальных сетей. Когда вы оставляете свою фотографию в Сети, вы оставляете возможность найти себя.
+You may have been confused by the "biometric data" point, but alas, its exploitation has long been part of our lives. Face searches, which use the same technology as that used to unlock a personal phone through the front camera, and extensive databases from social media have long been widespread. When you leave your photos on the Web, you leave the opportunity to a hostile threat actor to find you.
 
-### Анализируем угрозы
+### Analyzing threats
 
-В соответствии с процессом OPSEC проанализируем, кто может быть заинтересован в подобных действиях и как может использовать информацию о нас. Давайте рассмотрим популярные случаи использования первичных данных на схеме ниже и попробуем сделать выводы. 
+According to the OPSEC process, let's analyse who might be interested in such actions and how information about us might be exploited. In the diagram below, let's look at popular uses of primary data and try to draw conclusions. 
 
-![Что можно сделать с нашими данными?](../img/0869d0a1e60173af48378.png)
+![What can be done with our data?](../img/0869d0a1e60173af48378.png)
 
-Что можно сделать с нашими данными?
-Наверняка вы обратили внимание, что большинство пунктов связано с **получением денег**. Думаю, финансовая мотивация мошенников не требует объяснения. Единственное, что следует отметить — деньги могут быть похищены не только у вас, но и у других лиц. В этом случае критичным для нас становится участие в уголовном процессе, куда могут привлечь как свидетелем, так и обвиняемым. 
+What can be done with our data?
+You have probably noticed that most entries are related to **gaining money**. I think the financial motivation of the scammers is self-explanatory. The only issue to note is that money can be stolen from you and others. В этом случае критичным для нас становится участие в уголовном процессе, куда могут привлечь как свидетелем, так и обвиняемым. 
```

Comment:
Let's use "It is critical for us to not participate in court case, cause we can be both a witness and a accused."

Reply:
That sound good to me!